tesla_ble_mqtt/config.yaml

```diff
@@ -17,33 +17,30 @@ map:
   - share:rw
 startup: services
 options:
-  vin1: ""
-  vin2: ""
-  vin3: ""
-  presence_detection: false
-  ble_mac1: ""
-  ble_mac2: ""
-  ble_mac3: ""
+  vin: ""
```

Comment:
Need to figure out if we keep vin and/or vin1 (ble_mac...) for older installation to ease migration, unless in the CHANGELOG we mention to save a copy prior to upgrade?

Reply:
Removed.